src/server/pps/server/api_server.go

```diff
@@ -1012,27 +1014,27 @@ func (a *apiServer) listDatum(pachClient *client.APIClient, job *pps.Job, page,
 		return 0, 0, goerr.New("getPageBounds: unreachable code")
 	}
 
-	df, err := workerpkg.NewDatumIterator(pachClient, jobInfo.Input)
+	dit, err := datum.NewIterator(pachClient, jobInfo.Input)
```

Comment:
Changed the datum iterator interface name because it's now in its own package, so `datum.Iterator` is a little clearer than `worker.DatumIterator`.  Also, many places in the code still used `df` (for `DatumFactory`), so I updated those to `dit`.

Reply:
So happy to see the word `Factory` used less in our codebase.